### DISCUSSION TITLE
Filter string attributes lexicographically using operators?

### DISCUSSION THREAD

**Discussion by underflow00** [meilisearch/product#795] [Feedback & Feature Proposal]
It would be good if we can filter string attributes lexicographically using operators, like `attribute > "abc"`.
Useful for pagination, as an alternative to using `offset`.

**Comment by macraig** [comment]
Thank you for taking the time to share your feedback. We agree that lexicographic string filtering is a valuable feature. While our current priorities are focused elsewhere, we’re definitely interested in supporting this in the future. In the meantime, I’ll keep this discussion open so other users can share their thoughts and upvote.

**Comment by macraig** [comment]
Hi @underflow00 , 

We have just released the first RC (release candidate) of Meilisearch containing a way to filter string attributes lexicographically using operators!

You can test it by using

- the [release assets](https://github.com/meilisearch/meilisearch/releases)
- the [Meilisearch Docker image](https://hub.docker.com/repository/docker/getmeili/meilisearch/general)

```
docker run -it --rm -p 7700:7700 -v $(pwd)/meili_data:/meili_data getmeili/meilisearch:v1.15.0-rc.0

```

You are welcome to leave your feedback in this discussion.

If you encounter any bugs, please report them [here](https://github.com/meilisearch/meilisearch/issues/new?assignees=&labels=&template=bug_report.md&title=).
Thanks in advance for your help and your involvement in Meilisearch ❤️

🎉 Official and stable release containing this change will be available on `June 9th`, `2025`

⚠️ RC (release candidates) are not recommended for production

**Comment by macraig** [comment]
Hello everyone 👋 

[v1.15](https://github.com/meilisearch/meilisearch/releases/tag/v1.15.0) has been released! 🦘   You can now filter string attributes lexicographically using operators. ✨
